### DISCUSSION TITLE
Understanding bind-utils installation in Dockerfile

### DISCUSSION THREAD

**Discussion by Boojapho** [strimzi/strimzi-kafka-operator#5039] [Q&A] [answered]
Currently, bind-utils is used in the strimzi/base image:  https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/base/Dockerfile#L8

I'm doing my own hardened build of the Strimzi images based on the dockerfiles here.  I'm trying to understand if I can remove bind-utils from some of the images.  Can anyone help me figure out what images use bind-utils and what images don't need it?

**Comment by scholzj** [comment, marked as answer]
First of all, the UBI8 images are used in the main branch only. They are not in any release. I do not remember anymore why is it needed. Probably because of nslookup or something like that? If you have any improvement, feel free to open a PR.

**Reply by Boojapho** [reply]
Fair enough for UBI8.  I see you use CentOS for the release branches.  But the bind-utils is still part of the release since the image is built on the strimzi/base image.

I'll be happy to open a PR, but I don't know enough about where or when the package is used.  I'll search around for nslookup or dig...

**Reply by Boojapho** [reply]
I think this might be the only place:
https://github.com/strimzi/strimzi-kafka-operator/blob/main/cluster-operator/scripts/cluster_operator_run.sh#L6
